addons/website_event/static/src/scss/event_templates_page.scss

```diff
@@ -34,24 +34,21 @@
         }
     }
 
-    .o_wevent_registration_btn {
-        flex: 0 0 100%;
+    .o_wevent_registration_title {
+        font-weight: $font-weight-bold;
+        text-transform: uppercase;
+        font-family: "Source Sans Pro", "Odoo Unicode Support Noto", sans-serif !important;
```

Comment:
We usually avoid hardcoding font-families as those come from the selected theme.

Reply:
Because I changed the type of this title (it was a button, now it is a text), I couldn't get to have the same typo as before. Do you have any other way for me to do so ?

Reply:
Simply don't do it IMO.
If it's a text, then it should be styled as a text according to the selected template.
So I would remove this font-family rule.